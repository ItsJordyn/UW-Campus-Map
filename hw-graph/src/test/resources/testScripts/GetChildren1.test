# GetChildren testing file
# Tests that the children nodes from a single node can be
# Printed out (If there is more than one child node).

# Create a graph
CreateGraph graph1

# Add three nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

#Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n2 n3 e2
AddEdge graph1 n3 n1 e3
AddEdge graph1 n1 n3 e4

# Print the nodes and the outgoing edges from n1
ListChildren graph1 n1