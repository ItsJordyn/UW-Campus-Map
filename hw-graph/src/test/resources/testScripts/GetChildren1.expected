# GetChildren testing file
# Tests that the children nodes from a single node can be
# Printed out (If there is more than one child node).

# Create a graph
created graph graph1

# Add three nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

#Add edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n2 to n3 in graph1
added edge e3 from n3 to n1 in graph1
added edge e4 from n1 to n3 in graph1

# Print the nodes and the outgoing edges from n1
the children of n1 in graph1 are: n2(e1) n3(e4)